modules/tracking/test/test_trackers.cpp

```diff
@@ -396,6 +398,29 @@ void TrackerTest::checkDataTest()
     endFrame = (int)bbs.size();
 }
 
+void TrackerTest::gtrMemoryTest()
+{
+  cv::Rect2d roi;
+  cv::Mat frame;
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
+  string folder = cvtest::TS::ptr()->get_data_path() + "/" + TRACKING_DIR + "/" + video + "/" + FOLDER_IMG;
```

Comment:
Please try `findDataFile`

Reply:
You mean instead of ```get_data_path()``` ?

Reply:
Instead of all the paths management. See examples by https://github.com/opencv/opencv/search?q=findDataFile&unscoped_q=findDataFile